runtime/src/chpl-init.c

```diff
@@ -315,6 +315,10 @@ void chpl_execute_module_deinit(c_fn_ptr deinitFun) {
   deinitFn();
 }
 
+// These are currently defined in 'modules/internal/ExportWrappers.chpl'.
+void chpl_libraryModuleLevelSetup(void);
+void chpl_libraryModuleLevelCleanup(void);
```

Comment:
Can you declare these as extern?  Or whatever the syntax is in C?

Reply:
Ah, I didn't because all functions are extern as default, so it'd be redundant (in one of my commits they originally had the `extern` tag).

Reply:
Up to you, but I'm not opposed to extern for clarity like we have for `chpl_deinitModules`